docs/usage/self-hosted-configuration.md

```diff
@@ -822,12 +822,6 @@ It could then be used in a repository config or preset like so:
 
 Secret names must start with an upper or lower case character and can have only characters, digits, or underscores.
 
-## skipInstalls
```

Comment:
looks like a wrong change in this PR

Reply:
This seems to come from v36 branch were this has been reverted.

Reply:
I cherry picked the revert commit from v36 and it was applied cleanly. No idea how it was missed before?!